tracing-flame/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "tracing-flame"
+version = "0.1.0"
+authors = ["Jane Lusby <jlusby@yaah.dev>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tracing-subscriber = { path = "../tracing-subscriber", version = "0.2.3", default-features = false, features = ["registry", "fmt", "smallvec"] }
```

Comment:
I don't think we should always enable the `smallvec` feature on `tracing-subscriber`. Instead, we should leave that up to the user. If we want to turn it on by default, we should add a default on feature flag to _this_ crate, like

```toml
[features]
default = ["smallvec"]
smallvec = ["tracing-subscriber/smallvec"]
```